.github/workflows/ci.yml

```diff
@@ -28,7 +28,7 @@ jobs:
       matrix:
         os: [ubuntu-20.04]
         python-version: [3.8]
-        toxenv: [py38, integration, quality, translations]
+        toxenv: [py38, integration, quality]
```

Comment:
If we're removing this from CI workflows, we should also:
1. Remove the `- name: Install translations dependencies` step from `ci.yml`.
2. Remove this test from `tox.ini`.
3. Remove translation-related commands from `Makefile` if they will no longer work.
4. Update the "i18n compatibility" in the README.

Reply:
Thank you @Agrendalath , you're right. But we're doing this in phases (or milestones). Keeping the old mechanism will leave a useless code, but we'll eventually remove it after testing the new one and ensuring that it didn't break anything

Reply:
@shadinaif, if the CI check fails with [KeyError: 'compilejsi18n'](https://github.com/openedx/xblock-drag-and-drop-v2/actions/runs/4405954433/jobs/7717972381), then the code I listed above is not _useless_, but _unusable_. Keeping it in the repo will be confusing for developers.

Reply:
I'm marking this as resolved because we removed all breaking changes. Also restored `translations` test